### PR TITLE
control_command switch to UDP, issue #227

### PR DESCRIPTION
**Pull request by wouterheerwegh**
Forced subscriber of control_command to be UDP instead of TCP.

### CODE REVIEW

**Comment by SijmenHuizenga** [comment]
Great job on finding a workaround to solve the wsl connection problem!

Unfortunately, it seems [upd support in ros is not amazing and rospy does not support udp at all](https://answers.ros.org/question/345412/ros-communication-over-udp/). So when working with a python publisher we would probably still run into the same problem?

Aside from that, I would prefer to have all topics use the same protocol. If the sim were ever to run in a place where udp performs different then tcp, some topics on tcp would performing different then others on udp. This would make debugging network problems more difficult.

In conclusion: before we merge this I would like us to investigate a bit more to see if we can find a root cause. In #227 I've written my ideas on it.
